lib/TPP/Dialect/Tpp/BufferizableOpInterfaceImpl.cpp

```diff
@@ -254,6 +302,12 @@ struct AddBufferizationInterface
 // Ternary
 //===----------------------------------------------------------------------===//
 
+static bool isZeroFilled(Value val) {
```

Comment:
We have a `isZero` somewhere, this should go there.

Avoid creating little static helpers where there's a more generic function available.